test-forms/sequoia-benefits_login.html

```diff
@@ -0,0 +1,27 @@
+<!-- https://secure.bswift.com/login-page/sequoiatech -->
+<form autocomplete="on" id="login" name="login" novalidate="" class="ng-pristine ng-invalid ng-touched"><!---->
+    <div class="row-two">
+        <label class="display-ib-soft mb-smallest" for="username">
+            <span>Username</span>
+        </label>
+        <input aria-describedby="name_error" aria-required="true" class="input-large input-wide-full ng-pristine ng-invalid ng-touched" formcontrolname="username" id="txtUserName" name="txtUserName" style="background-color: rgb(255, 255, 255);" type="text" tabindex="0" data-manual-scoring="username">
```

Comment:
You removed the background from the input in `lakeshorelearning_checkout.html` — would it be worth doing that preemptively here as well?

(There's a few other `style="background-color …"` attributes below)

Reply:
That's fine. I removed the other one because while testing on a real browser it was showing the brackground we show after autofilling (sort of light yellow bg). I had probably copied the html from our browser and I forgot to remove that specific style. Removing that made it simpler to visually scan the page. Nothing to worry about. The background here is white, so it's inconsequential. Most likely no-one will ever look at this stuff in a real browser ever again 🙂.